py-utils/src/utils/msg_bus/message_bus/utils.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import time
+from datetime import datetime, timedelta
+
+logging.basicConfig(filename='log_file.log', level=logging.DEBUG)

```

Comment:
Please use cortx-utils logger. By default no logs should be generated only the debug logs.